packages/itwinui-react/src/core/ComboBox/ComboBox.test.tsx

```diff
@@ -655,14 +655,13 @@ it('should call onExpand and onCollapse when dropdown is opened and closed', asy
 
   const input = container.querySelector('input') as HTMLElement;
   await userEvent.click(input);
-  const list = document.querySelector('.iui-menu') as HTMLElement;
-  expect(list).toBeVisible();
+  expect(document.querySelector('.iui-menu')).toBeVisible();
   expect(onExpand).toHaveBeenCalled();
 
   const menuItem = document.querySelector('.iui-list-item') as HTMLElement;
 
   await userEvent.click(menuItem);
-  expect(list).not.toBeVisible();
+  expect(document.querySelector('.iui-menu')).toBeFalsy();
```

Comment:
nit: You used `toBeVisible()` on L658. Did you mean to use `.not.toBeVisible()` here?

Reply:
i think `not.toBeVisible()` doesn't always work because we remove this element from the dom. so `not.toBeFalsy()` is more reliable